app/src/pages/react-uswds-hello.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Alert,
+  Button,
+  Grid,
+  GridContainer,
+  Icon,
+} from "@trussworks/react-uswds";
+import Link from "next/link";
+import React from "react";
+
+/** Example page showing use of react-uswds components. Can be deleted once you're on a roll. */
+const HelloWorld = () => {
+  return (
+    <GridContainer>
+      <Grid row gap={5}>
+        <Alert type="success" headingLevel="h1" heading="Hello world">
+          This is an alert
+        </Alert>
+      </Grid>
+      <Grid row>
+        <Grid col="auto">
+          <Icon.Build size={9} />
+        </Grid>
+        <Grid col="auto">
+          <p>
+            Here is some example text. You can learn more{" "}
+            <Link href="https://trussworks.github.io/react-uswds/">
+              in the docs
+            </Link>
+            .
+          </p>
+        </Grid>
+      </Grid>
+      <Grid row gap={5}>
```

Comment:
Similar comment here about adding a column. I think that'll also fix the left alignment:

![CleanShot 2022-12-08 at 16 16 55@2x](https://user-images.githubusercontent.com/371943/206593882-3b7addc7-8495-4e0e-9498-0c931621f5de.png)